exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -43,28 +43,23 @@ public string CreateRockQuestion(int index)
 
         public bool IsPlayable()
         {
-            return (HowManyPlayers() >= MinPlayers);
+            return (players.Count >= MinPlayers);
```

Comment:
I like the way you have removed a semi hard coded variable - HowManyPlayers and used Count as this variable could change in future